lib/prefix.h

```diff
@@ -320,6 +320,7 @@ union prefixconstptr {
 
 /* Maximum string length of the result of prefix2str */
 #define PREFIX_STRLEN 80
+#define IPV4_PREFIX_STRLEN 19
```

Comment:
just use PREFIX_STRLEN.  I see no point in introducing another define especially since prefix2str is being used.

Reply:
Done.